server/config.cpp

```diff
@@ -464,6 +466,7 @@ bool Config::saveConfig() const {
     xml::newBoolChild(curNode, "LessExpLoss", lessExpLoss);
     xml::newBoolChild(curNode, "LogDeath", logDeath);
     xml::newBoolChild(curNode, "PkillInCombatDisabled", pkillInCombatDisabled);
+    xml::newBoolChild(curNode, "PlayersCanMudmailStaff", playersCanMudmailStaff);
```

Comment:
This needs a variable defined in Config.hpp and a default value set in the constructor.